servo-tidy.toml

```diff
@@ -4,7 +4,7 @@ skip-check-licenses = false
 
 [ignore]
 # Ignored packages with duplicated versions
-packages = ["lazy_static"]
+packages = ["lazy_static", "bitflags"]
```

Comment:
It was because `mp4parse-rust` depended on rust-cheddar. It's not the case anymore so I remove it.